doc/jvm/xgboost4j_spark_tutorial.rst

```diff
@@ -129,7 +129,7 @@ labels. A DataFrame like this (containing vector-represented features and numeri
 
 .. note::
 
-  There is no need to assemble feature columns from version 1.6.0+. Instead, users can specify an array of
+  There is no need to assemble feature columns from version 1.6.1+. Instead, users can specify an array of
   feture column names by ``setFeaturesCol(value: Array[String])`` and XGBoost4j-Spark will do it.
```

Comment:
typo: "feture" -> "feature"

Reply:
also, unrelated, but it would be helpful to comment whether is it possible to use the new categorical feature support in 1.6.X from xgboost4j.

Reply:
Thank you for the suggestion. It's Python only at the moment and is considered experimental. We will continue to expand it.